scripts/submit_skims.sh

```diff
@@ -629,18 +629,18 @@ MC_MAP=(
     ["_WZZ"]="-n 10 -x 0.057 -q short"
     ["_ZZZ"]="-n 10 -x 0.0147 -q short"
```

Comment:
Here and above: these are most likely negligible, all those samples would have no corrections if they're not genmatched to a Z/H. I suppose di/triboson samples can by "DYlike", as well as ZHToTauTau. For the rest I guess the fatjet will most likely be fake though

Reply:
updated!